src/Browsershot.php

```diff
@@ -411,6 +419,14 @@ protected function createCommand(string $url, string $action, array $options = [
             $command['options']['args'] = ['--no-sandbox'];
         }
 
+        if ($this->proxy) {
+            if (isset($command['options']['args'])) {
+                $command['options']['args'][] = ['--proxy-server='.$this->proxy];
+            } else {
```

Comment:
Could you avoid using an else statement? Use a helper function and early returns.

Reply:
I updated it to use a helper function to build the args array in the latest revision. The if part was incorrect anyway as it added an array instead of a string to the opts array.